webrtc-stats.html

```diff
@@ -946,8 +953,8 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  Total number of RTP packets received for this <a>SSRC</a>. At the receiving endpoint,
-                  this is calculated as defined in [[!RFC3550]] section 6.4.1. At the sending
+                  Total number of RTP packets received for this <a>SSRC</a>. This includes retransmissions.
+                  At the receiving endpoint, this is calculated as defined in [[!RFC3550]] section 6.4.1. At the sending
```

Comment:
Somewhat related, a lot of this language is pretty confusing since, due to inheritance, sometimes it relates to what is sent/received via RTCP. However, there are local counters too (you can count what you send/receive and that should actually be the basis for what you put into RTCP SR/RR)

Reply:
Typically the stats reported in RTCP are snapshots of the running counters.

For example jitter is calculated when each RTP packet is received. The algorithm in 3550 should apply irrespective. However depending on the timestamp in JSON along with the type of report explains if the data is from the most recent packet received or most recent RTCP.

What I think may be makes sense to do is that we explain this somewhere and then refer to the explanation section to avoid confusion.

Reply:
Yeah. But for example [packetsReceived](https://w3c.github.io/webrtc-stats/#dom-rtcreceivedrtpstreamstats-packetsreceived) should be based on a counter when it is on inbound-rtp but based on RTCP on remote-inbound-rtp. The inheritance makes it weird, describing it separately might help.